wallet/wallet.go

```diff
@@ -74,3 +74,24 @@ func (m *Module) Balance(ctx context.Context, addr string) (uint64, error) {
 	}
 	return b.Uint64(), nil
 }
+
+// SendFil sends fil from one address to another
+func (m *Module) SendFil(ctx context.Context, from string, to string, amount *big.Int) error {
+	f, err := address.NewFromString(from)
+	if err != nil {
+		return err
+	}
+	t, err := address.NewFromString(to)
+	if err != nil {
+		return err
+	}
+	msg := &types.Message{
+		From:     f,
+		To:       t,
+		Value:    types.BigInt{Int: amount},
+		GasLimit: 1000, // ToDo: how to handle gas?
+		GasPrice: types.NewInt(0),
```

Comment:
Not sure if now is the time or place to deal with gas. I see there is a method to estimate gas, but I don't really understand the scope of this for now.

Reply:
Last time I read about that, was still a ToDo in Lotus.
Better to leave this as it is until we know the correct way to do that... good default to be 0.

Reply:
👍